lms/djangoapps/courseware/field_overrides.py

```diff
@@ -70,6 +69,12 @@ def wrap(cls, user, wrapped):
         any performance impact of this feature if no override providers are
         configured.
         """
+        # if ccx not enabled for this course, just return unwrapped
+        from courseware.courses import get_course_by_id
+        course = get_course_by_id(course_id)
+        if not course.enable_ccx:
+            return wrapped
```

Comment:
@cpennington: so the suggestion then is to pass the loaded course in to the get_module_* methods and chain it down to where we'll need it.  That works.  Agreed that adding more parameters is not great, but perhaps there could be a trade of passing in course rather than course_id since one can be derived from the other?